app/src/keymap.c

```diff
@@ -247,12 +257,27 @@ int zmk_keymap_set_layer_binding_at_idx(zmk_keymap_layer_id_t layer_id, uint8_t
 
     ASSERT_LAYER_VAL(layer_id, -EINVAL)
 
+    uint32_t *pos_map;
+    int ret = zmk_physical_layouts_get_selected_to_stock_position_map(&pos_map);
+    if (ret < 0) {
```

Comment:
This case isn't currently possible. Will it ever be possible?

Reply:
Rather than change the API later, I wanted to ensure we guard now in case things change later with how position maps get created.